README.md

```diff
@@ -1,221 +1,274 @@
+  "clientSecretUrl": "https://control-eastus.identity.azure.net/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myresourcegroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myidentity/credentials?tid=00000000-0000-0000-0000-000000000000&oid=00000000-0000-0000-0000-000000000000&aid=00000000-0000-0000-0000-000000000000",
+  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myresourcegroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myidentity",
+  "location": "eastus",
+  "name": "myidentity",
+  "principalId": "00000000-0000-0000-0000-000000000000",
+  "resourceGroup": "myresourcegroup",
+  "tags": {},
+  "tenantId": "00000000-0000-0000-0000-000000000000",
+  "type": "Microsoft.ManagedIdentity/userAssignedIdentities"
+}
 ```
 
-### Install User Azure Identity on k8s cluster 
-
-Edit and save this as aadpodidentity.yaml
+### 3. Install the Azure Identity
 
-Set `type: 0` for User Assigned MSI; `type: 1` for Service Principal
+Save this Kubernetes manifest to a file named `aadpodidentity.yaml`:
 
-```
+```yaml
 apiVersion: "aadpodidentity.k8s.io/v1"
 kind: AzureIdentity
 metadata:
- name: <a-idname>
+  name: <a-idname>
 spec:
- type: 0
- ResourceID: /subscriptions/<subid>/resourcegroups/<resourcegroup>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<managedidentity-resourcename>
- ClientID: <clientid>
+  type: 0
+  ResourceID: /subscriptions/<subid>/resourcegroups/<resourcegroup>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<name>
+  ClientID: <clientId>
 ```
 
-```
+Replace the placeholders with your user identity values. Set `type: 0` for user-assigned MSI or `type: 1` for Service Principal.
+
+Finally, save your changes to the file, then create the `AzureIdentity` resource in your cluster:
+
+```shell
 kubectl create -f aadpodidentity.yaml
 ```
 
-### Understanding Namespaced identities
-The system will match `pod` to `identity` across namespaces by default. This behavior can be modified to match to pods within the namespace that holds `AzureIdentity` by
+### 4. (Optional) Match Pods in the Namespace
 
-1. On Azure Identity basis, by adding `aadpodidentity.k8s.io/Behavior: namespaced` annotation (You have to add the annotation on each `AzureIdentity` you want to apply this behavior on).
-2. Default namespaced behavior on all identities by adding `--forceNamespaced=true` argument on the command line  or declare `FORCENAMESPACED=true` environment variable (for both `nmi` and `mic`).
+By default, AAD Pod Identity matches pods to identities across namespaces. To match only pods in the namespace containing `AzureIdentity`, use one of these techniques:
 
+* Attach a `aadpodidentity.k8s.io/Behavior: namespaced` [annotation] to each `AzureIdentity` resource.
+* Add the `--forceNamespaced=true` command line argument or set the `FORCENAMESPACED=true` environment variable when starting both the MIC and NMI components.
```

Comment:
would be helpful to see an example yaml of this

Reply:
Good idea, I'll add something.